llm/rag/serve_rag.yaml

```
@@ -0,0 +1,80 @@
+name: serve-legal-rag
+
+workdir: .
+
+resources:
+  accelerators: 
+    L4: 4
+    L40S: 4
```

minor: this might be simpler
```suggestion
  accelerators: {L4:4, L40S:4}
```